<launch>
	<!-- Launch sensor launch file in usv_sensor package/-->
	<include file="$(find usv_sensor)/launch/usv_sensor_on.launch"/>
	<node name="auto_pwm" pkg="uri_usv" type="auto_pwm" >
	</node>
	<node name="heading_PID" pkg="uri_usv" type="heading_PID">
		<param name = "K_p"  value ="40.0"/>
		<param name = "K_i"   value ="0.01"/>
		<param name = "K_d"   value ="20.0"/>
		<param name = "K_g"   value ="0.001"/>
		<param name = "Total_thrust" value ="1.6"/>
	</node>
     	<!--guidance  /-->
	<node name="guidance_gps_node" pkg="uri_usv" type="guidance_gps_node"  output="screen">
		<remap from="/c_heading" to="/control/heading" />
   		<rosparam param="lat">[21.311032938133,21.311931867658]</rosparam>
    		<rosparam param="lon">[-157.890086266199,-157.88896389019]</rosparam>
	</node>
</launch>
